airbyte-integrations/connectors/source-greenhouse/pyproject.toml

```diff
@@ -16,8 +16,8 @@ repository = "https://github.com/airbytehq/airbyte"
 include = "source_greenhouse"
 
 [tool.poetry.dependencies]
-python = "^3.9,<3.12"
-airbyte-cdk = "0.80.0"
+python = "^3.10,<3.12"
+airbyte-cdk = "^6"
 dataclasses-jsonschema = "==2.15.1"
```

Comment:
Can you please update pytest requirement to 8.0+?

Reply:
updated